src/mesh/generated/meshtastic/deviceonly.pb.h

```diff
@@ -33,6 +33,8 @@ typedef struct _meshtastic_PositionLite {
     uint32_t time;
     /* TODO: REPLACE */
     meshtastic_Position_LocSource location_source;
+    /* Indicates the bits of precision set by the sending node */
```

Comment:
For reference, this code is automatically generated. The source is here: https://github.com/meshtastic/protobufs

Reply:
Thanks, I'm aware. If this PR is approved, I'll make PR for that repo as well before merging this.